### PR TITLE
chore: change ct -> component, projectType -> testingType everywhere for consistency

### PR DESCRIPTION
**Pull request by lmiller1990**
- Closes N/A

### User facing changelog

- Change `'ct'` to `'component'` throughout the code base for consistency
- Change `projectType` to `testingType` for consistency

This makes it much easier to understand what's going on - no need to have multiple ways to refer to the same thing.

### Additional details

It's very difficult to know when to use `ct` or `component`. Let's standardize and use `'component'` to represent component testing.

### How has the user experience changed?

N/A

### PR Tasks

- [x] Have tests been added/updated?
- [ ] Has the original issue or this PR been tagged with a release in ZenHub? 
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [ ] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

File: packages/server/lib/project-base.ts [hunk 1]
```diff
@@ -83,18 +83,18 @@ export class ProjectBase<TServer extends ServerE2E | ServerCt> extends EE {
   private _recordTests?: any = null
 
   public browser: any
-  public projectType: RunnerType
+  public testingType: RunnerType
```
Comment (elevatebart) [inline packages/server/lib/project-base.ts:86]: Why not rename the type as well here? Too much work?

**Review by elevatebart — DISMISSED** [review]
Can you also rename RunnerType in `packages/server/lib/specs-store.ts` into TestingType or use the Cypress.d.ts type?

This already a leap forward. 
👍

File: packages/server/lib/project-base.ts (hunk 1 above)
Comment (lmiller1990) [reply packages/server/lib/project-base.ts]: Good idea, done. I just did find and replace on `testingType` - that's why I missed `RunnerType`. It's now using `Cypress.TestingType`.

**Comment by lmiller1990** [comment]
@elevatebart good review, I made those changes... this small change makes things a lot easier to reason about.

**Review by elevatebart — APPROVED** [review]

**Pull request merged**